controllers/dataprotection/backup_controller.go

```diff
@@ -397,6 +415,17 @@ func (r *BackupReconciler) patchBackupObjectMeta(
 			return false, err
 		}
 		request.Labels[dptypes.ClusterUIDLabelKey] = string(cluster.UID)
+		backupType := dputils.GetBackupType(request.ActionSet, request.BackupMethod.SnapshotVolumes)
```

Comment:
I recommend encapsulating a separate function to set cluster password annotation,  corresponds to setClusterSnapshotAnnotation. FYI.